test/integration/integration_test.py

```diff
@@ -0,0 +1,144 @@
+## The integration tests consists of a back and forth between python (which we use to create and validate
+## image queries) and bash (which we use to check deployments are properly rolled out)
+## This file contains all the modes that we use for integeration testing.
+## Modes:
+## - Create the integration test detector
+## - Submit the initial dog/cat image query to the edge, expect low confidence
+## - Train the edge model by submitting image queries to the cloud.
+## - Submit the final dog/cat image query to the edge, expect high confidence
+
+import argparse
+import random
+import time
+
+from groundlight import Groundlight, GroundlightClientError
+from model import Detector
+
+NUM_IQS_TO_IMPROVE_MODEL = 10
+ACCETABLE_TRAINED_CONFIDENCE = 0.8
+
+
+def get_groundlight():
+    try:
+        return Groundlight(endpoint="http://localhost:30107")
+    except GroundlightClientError:
+        # we use this to create a detector since we do that before setting up edge
+        # although maybe we want to be more careful here about making sure that's
+        # the case we're in
+        return Groundlight()
+
+
+gl = get_groundlight()
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Submit a dog and cat image to k3s Groundlight edge-endpoint for integration tests"
+    )
+    parser.add_argument(
+        "-m",
+        "--mode",
+        type=str,
+        choices=["create_detector", "initial", "improve_model", "final"],
+        help="Mode of operation: 'initial', 'many', or 'final'",
```

Comment:
Should this have "create_detector", "initial", "improve_model", and "final" as the options? Or is this saying something different?